en/contributing/test.md

```diff
@@ -1,20 +1,28 @@
 # Testing
 
-## Unit Tests
 
-To run the unit tests:
+*DroneCore* has both unit and integration tests, written using the *Google C++ Test Framework*. 
+The tests are run every time new code is committed to our codelines, and must pass before the code can be merged.
```

Comment:
Currently that's only true for unit tests.

Reply:
OK. I'll update